### PR TITLE
[Refactor] remove redundant class

### PR DESCRIPTION
**Pull request by mproch**

### CODE REVIEW

File: engine/flink/management/src/main/scala/pl/touk/nussknacker/engine/management/FlinkRestManager.scala [hunk 1]
```diff
@@ -206,21 +206,21 @@ class FlinkRestManager(config: FlinkConfig, modelData: ModelData, mainClassName:
 
   override protected def cancel(job: ProcessState): Future[Unit] = {
     basicRequest
-      .patch(flinkUrl.path("jobs", job.deploymentId.value))
+      .patch(flinkUrl.path("jobs", job.deploymentId.get.value))
```
Comment (lciolecki) [inline engine/flink/management/src/main/scala/pl/touk/nussknacker/engine/management/FlinkRestManager.scala:209]: Is it good way..? In fact, is there possible situation when `job.deploymentId` is not defined?

File: ui/restmodel/src/main/scala/pl/touk/nussknacker/restmodel/displayedgraph/ProcessStatus.scala [hunk 2]
```diff
@@ -1,121 +1,86 @@
...
 import java.net.URI
-import io.circe.generic.JsonCodec
-import io.circe.Json
-import pl.touk.nussknacker.engine.api.deployment.ProcessActionType.ProcessActionType
 import pl.touk.nussknacker.engine.api.deployment.simple.{SimpleProcessStateDefinitionManager, SimpleStateStatus}
 import pl.touk.nussknacker.engine.api.deployment.{ProcessState, ProcessStateDefinitionManager, StateStatus}
-import pl.touk.nussknacker.restmodel.codecs.URICodecs.{uriDecoder, uriEncoder}
-
-//TODO: Do we really  we need ProcessStatus and ProcessState - Do these DTO's do the same things?
-@JsonCodec case class ProcessStatus(status: StateStatus,
```
Comment (lciolecki) [inline ui/restmodel/src/main/scala/pl/touk/nussknacker/restmodel/displayedgraph/ProcessStatus.scala:12]: Yupi! :)

File: ui/server/src/main/scala/pl/touk/nussknacker/ui/api/AppResources.scala [hunk 3]
```diff
@@ -126,13 +127,13 @@ class AppResources(config: Config,
...
     }
 
-  private def notRunningProcessesThatShouldRun(implicit ec: ExecutionContext, user: LoggedUser): Future[Map[String, Option[ProcessStatus]]] = {
+  private def notRunningProcessesThatShouldRun(implicit ec: ExecutionContext, user: LoggedUser): Future[Map[String, Option[ProcessState]]] = {
     for {
       processes <- processRepository.fetchDeployedProcessesDetails[Unit]()
       statusMap <- Future.sequence(statusList(processes)).map(_.toMap)
     } yield {
       statusMap.filterNot{
-        case (_, status) => status.exists(st => st.status.isDuringDeploy || st.status.isRunning)
```
Comment (lciolecki) [inline ui/server/src/main/scala/pl/touk/nussknacker/ui/api/AppResources.scala:135]: Beautiful ;)

**Review by nadberezny — APPROVED** [review]

**Review by lciolecki — APPROVED** [review]
LGTM, but I don't like this `.get.value` on deploymentId`.

File: engine/flink/management/src/main/scala/pl/touk/nussknacker/engine/management/FlinkRestManager.scala (hunk 1 above)
Comment (mproch) [reply engine/flink/management/src/main/scala/pl/touk/nussknacker/engine/management/FlinkRestManager.scala]: :+1: I fixed it

**Pull request merged**